src/components/image/index.tsx

```diff
@@ -112,6 +117,21 @@ export function Image({
     }
   }, [data, fetchingFileInProgress, imgError]);
 
+  const handleDelete = async () => {
+    if (!propThing || !solidDataset || !propProperty || !value) return;
+    try {
+      const updatedThing = removeUrl(propThing, propProperty, value as string);
```

Comment:
If the preceding `if` was changed so that it tests `typeof value === 'string'` rather than `!value`, would the type assertions still be needed ?